### PR TITLE
Support for GEL theming in the docz app

### PR DESCRIPTION
**Pull request by jossmac**
- delete media-queries package
- express breakpoints as brand token
- move facepaint logic to grid package
- move applicable deps to `peerDependencies`, e.g. "prop-types", "react", "react-dom"
- brands only have default export object
- add docs for tabcordion
- improve docs for icon

### CODE REVIEW

File: components/core/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
 		"@emotion/core": "^10.0.4",
 		"@emotion/styled": "^10.0.4",
 		"emotion-theming": "^10.0.4",
+		"prop-types": "^15.6.2",
```
Comment (dominikwilkowski) [inline components/core/package.json]: should also be peer

File: components/media-queries/package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
 	},
 	"main": "dist/index.js",
 	"dependencies": {
-		"facepaint": "^1.2.1"
+		"facepaint": "^1.2.1",
+		"prop-types": "^15.6.2"
```
Comment (dominikwilkowski) [inline components/media-queries/package.json]: should be peer

File: package.json [hunk 3]
```diff
@@ -71,6 +71,7 @@
 		"emotion-theming": "^10.0.4",
 		"facepaint": "^1.2.1",
 		"prettier": "^1.15.3",
+		"prop-types": "^15.6.2",
```
Comment (dominikwilkowski) [inline package.json]: peer?

File: components/media-queries/package.json (hunk 2 above)
Comment (jossmac) [reply components/media-queries/package.json]: will delete this package monday

File: components/core/package.json (hunk 1 above)
Comment (jossmac) [reply components/core/package.json]: fixed

File: components/media-queries/package.json (hunk 2 above)
Comment (dominikwilkowski) [reply components/media-queries/package.json]: @jossmac deleted?

Comment (jossmac) [reply components/media-queries/package.json]: yes

**Pull request merged**